paymob/accept/connection.py

```diff
@@ -41,32 +44,34 @@ def _get_headers(self) -> Dict[str, Any]:
             "Authorization": f"{self.auth_token}",
         }
 
-    def _get_auth_token(self) -> str:
+    def _get_auth_token(self) -> Union[str, None]:
         """Retrieve an Auth Token
 
         Returns:
-            str: Auth Token
+            Union[str, None]: Auth Token
         """
 
         request_body = {"api_key": Credentials.ACCEPT_API_KEY}
 
-        _, data, _ = self.post(
-            url=URLsConfig.AUTH_TOKEN_URL,
```

Comment:
Why rename it from  `AUTH_TOKEN_URL` to `AUTH_TOKEN` becuase I think that `AUTH_TOKEN_URL` is more meaningful name

Reply:
The naming of `URLsConfig` class already indicates that it has only `URLs` so I think we don't need to attach `_URL` at the end of every `URL` variable

Reply:
ok got it